adops/step-by-step.md

```diff
@@ -21,10 +20,22 @@ Because integrating with Prebid could mean having to create thousands of line it
 {: .alert.alert-success :}
 Prebid provides a script you can use to automate these steps: [Prebid Line Item Manager](/tools/line-item-manager.html).
 
+{: .alert.alert-info :}
+Note that running "outstream" video ads requires special attention if you're a publisher
+that utilizes both Prebid.js and Prebid Mobile. See the [planning guide note](/adops/adops-planning-guide.html#rendering-mobile-outstream-ads) for more background.
```

Comment:
not if you have a player that can play outstream. Maybe the right thing is to force publishers to use a valid outstream player; it could reduce complexity.

Reply:
I think you may be missing the root problem here -- it's that GAM gets confused and serves the wrong line item unless there's special targeting or we use the size hack.